background.js

```diff
@@ -707,6 +707,45 @@ browser.compose.onBeforeSend.addListener(tab => {
   }
 });
 
+browser.runtime.onMessageExternal.addListener(externalListener);
+
+function externalListener(message, sender, sendResponse) {
+  if (message["action"] == "getPreferences") {
```

Comment:
Not that it matters here, but I've been trying to stick with the stricter `===` comparison. Same with a few spots below.

Reply:
Will fix.